src/select-menu/src/SelectMenuContent.js

```diff
@@ -24,7 +24,7 @@ const DefaultTitleView = ({ close, headerHeight, title }) => (
     <IconButton
       icon={CrossIcon}
       appearance="minimal"
-      height={24}
+      size="small"
```

Comment:
Should we decouple this from the actual theme?

Reply:
Hm... curious what you mean?

Reply:
We could use `height` internally, since we dont know what's in the theme.

Reply:
Yeah, think we should use `height` directly instead. This doesn't need to respect the theme imo.